webview/src/shared/variables.scss

```diff
@@ -15,6 +15,7 @@ $row-bg-color: $bg-color;
 $header-fg-color: $fg-color;
 $row-bg-alt-color: var(--vscode-sideBar-background);
 $row-bg-selected-color: var(--vscode-list-activeSelectionBackground);
+$row-fg-selected-color: var(--vscode-list-activeSelectionForeground);
```

Comment:
From what I can tell, this variable is always set in all themes so we don't need to a fallback like we did with the icon color.